index.ts

```diff
@@ -0,0 +1,9 @@
+export function orderRequest(): string {
+    return "Made Order"
+}
+
+export function cancelOrder(id: number): string {
+    return "Cancelled Order with ID: " + id
```

Comment:
The template string might be more clear. 
`return `Cancelled Order with ID: ${id}``

Reply:
Sure thing